Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -339,15 +338,19 @@
   - LAWYERS
   - LIBRARIANS
   - MEDICAL DOCTORS
+  - MONKEYS
+  - NERDS
+  - NON-CREW
   - MIMES
   - PARAMEDICS
   - PASSENGERS
   - QUARTERMASTERS
   - RESEARCH DIRECTORS
-  - ROBOTICISTS
```

Comment:
i think its nice to leave it in game

Reply:
right now they arent a job, but yeah makes sense

Reply:
its arguably the richest part of sci